readme.md

```diff
@@ -19,21 +19,21 @@ SFML is a simple, fast, cross-platform and object-oriented multimedia API. It pr
 
 ## Download
 
-You can get the latest official release on [SFML's website](http://www.sfml-dev.org/download.php). You can also get the current development version from the [Git repository](https://github.com/SFML/SFML).
+You can get the latest official release on [SFML's website](https://www.sfml-dev.org/download.php). You can also get the current development version from the [Git repository](httpss://github.com/SFML/SFML).
```

Comment:
Interesting protocol you got there. 😉

Reply:
I ran one command twice... I knew I would miss something 😄